packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts

```diff
@@ -19,7 +19,7 @@ export const getStyles = (props: IFabricStyleProps): IFabricStyles => {
   return {
     root: [
       classNames.root,
-      isFocusVisible && 'is-focusVisible',
+      isFocusVisible && 'is-focusVisible ms-Fabric--isFocusVisible',
```

Comment:
Note that without this, child windows were not receiving this class even when wrapped in a `Fabric` element, as it was only being added to `document.body`